packages/bpk-component-page-indicator/README.md

```diff
@@ -0,0 +1,38 @@
+# bpk-component-page-indicator
+
+> Backpack example component.
+
+## Installation
+
+```sh
+npm install bpk-component-page-indicator --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkPageIndicator from 'bpk-component-page-indicator';
+
+export default () =>
+  <BpkPageIndicator
+    currentItem={0}
+    totalItems={7}
+    updateItemCallback={(newIndex) => {
+      console.log('Slide to new index:', newIndex);
+    }}
+  />;
+```
+
+## Props
+
+| Property            | PropType | Required | Default Value |
+|---------------------|----------|----------|---------------|
+| currentItem         | number   | true     | -             |
+| totalItems          | number   | true     | -             |
+| className           | string   | false    | null          |
+| itemClassName       | string   | false    | null          |
+| activeItemClassName | string   | false    | null          |
+| ariaLabel           | string   | false    | 'Go to Slide' |
+| showNav             | bool     | false    | false         |
+| updateItemCallback  | func     | false    | null          |
```

Comment:
Would be better to update to `onClick: (event, index, direction)` instead, so that we can use it to response the three actions, clicking on the bullet or clicking on the left and right buttons

Reply:
And I don't think we need `itemClassName` and `activeItemClassName`, because we don't expect the bullet's style to change, that's one reason why it's wrapped as a backpck component, giving users a consistent product experience, which of course includes its look visual

Reply:
Have replaced updateItemCallback with onClick and add direction(on prev button -> -1, on next button -> 1, on bullets -> 0)

Reply:
Then you need to update some usages as well